test/integration/dialects/mysql/query-generator.test.js

```diff
@@ -369,6 +369,11 @@ if (Support.dialectIsMySQL()) {
           arguments: ['myTable', {where: {field: {ne: null}}}],
           expectation: 'SELECT * FROM `myTable` WHERE `myTable`.`field` IS NOT NULL;',
           context: QueryGenerator
+        }, {
```

Comment:
@janmeier can't be a where test, since it only happens for raw substitions is my guess.